### DISCUSSION TITLE
Virtual Conference Platform

### DISCUSSION THREAD

**Discussion by brandon-jernigan** [resbazaz/festival2024#22] [Announcements]
Hello all! Val, Heidi and I discussed the virtual platform for ResBaz at hacky hour, and it looks like Gather Town could cost around $300 for a well-attended virtual day. We can use Zoom for free, most people are comfortable with Zoom and know what to expect, and it lends itself to workshops and presentations. So we think we'll just use Zoom this time. Any other thoughts on this?

**Comment by lwdozal** [comment]
I think zoom works!

**Comment by ghost** [comment]
didn't realize we would have to pay for gather! zoom sounds good then

**Reply by julianpistorius** [reply]
I think we could get sponsorship to pay for the Gather if people want.

**Reply by julianpistorius** [reply]
I don't know why I haven't looked for this before, but there is at least one open-source, self-hosted alternative to gather.town:

https://github.com/workadventure/workadventure

https://workadventu.re/

**Reply by julianpistorius** [reply]
I don't want to complicate things, so if people are OK with Zoom, let's go with that.
